pages/linux/pidof.md

```diff
@@ -0,0 +1,19 @@
+# pidof
+
+> Gets the id of a process using it's name.
```

Comment:
```suggestion
> Gets the ID of a process using its name.
```

id should be changed to ID in all other instances, too.

Reply:
Is there a rule against using apostrophes? I think my grammar is correct. Also how should I handle having back ticks ( ` ) inside a code block?

Reply:
About back ticks, I am not sure. /cc @agnivade @pxgamer

Here it is `its` because that word represents a pronoun. You can search for `its vs it's` on the Internet to find when to use each of them.

Reply:
`it's`/`its` is a special case. While one usually includes an apostophe to indicate possessiveness, you in fact omit it with `its`. The apostrophe in this specific instance indicates a contraction of `it is`.

Also, `id` → `ID` doesn't feel required to me. It could be either - it doesn't really matter.

Reply:
It's a norm to write `ID` uppercase, as it's an acronym. `id` is a term from psychoanalysis and is unrelated to tech.